modules/data.remote/R/call_remote_process.R

```diff
@@ -0,0 +1,400 @@
+##' call the Python - remote_process from PEcAn and store the output in BETY
+##' 
+##' @name call_remote_process
+##' @title call_remote_process
+##' @export
+##' @param settings PEcAn settings list containing remotedata tags: siteid, sitename, raw_mimetype, raw_formatname, geofile, outdir, start, end, source, collection, scale, projection, qc, algorithm, credfile, pro_mimetype, pro_formatname, out_get_data, out_process_data
+##' @examples
+##' \dontrun{
+##' call_remote_process(settings)
+##' }
+##' @author Ayush Prasad
+##'
+
+call_remote_process <- function(settings){
+  
+  # information about the date variables used in call_remote_process -
+  # req_start, req_end : start, end dates requested by the user, the user does not be aware about the status of the requested file in the DB  
+  # start, end : effective start, end dates created after checking the DB status. These dates are sent to remote_process for downloading and processing data
+  # write_raw_start, write_raw_end : start, end dates which are used while inserting and updating the DB
+  # the "pro" version of these variables have the same meaning and are used to refer to the processed file
+
+  reticulate::import_from_path("remote_process", file.path("..", "inst"))
```

Comment:
This is wrong and will not work from when being called from anywhere else but the only other way I have been able to get this work is by directly specifying the absolute path `/pecan/modules/data.remote/inst` which I think is also not appropriate? Using `file.path(system.file("remote_process.py", package = "PEcAn.data.remote"))` gives me a module not found error. I have no idea why.